portfolio/src/main/webapp/aboutme.html

```diff
@@ -3,54 +3,63 @@
-                The endless possibilities and the power of a couple of lines of code led me to a different 
-                type of puzzle.</p> 
-            <p>The word hacking has become infamous, too many people think of crime when they hear the word; while in reality, 
-                hacking is simply the art of creativity. It is the art of solving puzzles that were not meant 
-                to be solved. Programming has taught me to learn to break things as it can only result 
-                in an improvement in the following iteration; this is why security leaders perform 
-                sophisticated adversarial attack simulations. Penetration testing has become a home for 
-                me as it provides me the ultimate puzzle that requires all my passions to solve; math, 
+                The endless possibilities and the power of a couple of lines of code led me to a different
+                type of puzzle.</p>
+            <p>The word hacking has become infamous, too many people think of crime when they hear the word; while in reality,
+                hacking is simply the art of creativity. It is the art of solving puzzles that were not meant
+                to be solved. Programming has taught me to learn to break things as it can only result
+                in an improvement in the following iteration; this is why security leaders perform
+                sophisticated adversarial attack simulations. Penetration testing has become a home for
+                me as it provides me the ultimate puzzle that requires all my passions to solve; math,
                 programming, and technical knowledge.</p>
-            
+
         </div>
-        
+
         <div id="footer">
             <div>
-                <a href="index.html"><button type="button" class="btn btn-outline-dark"><span class="glyphicon glyphicon-menu-left"></span> BACK</button></a>
-                <button type="button" onclick="getComments()" class="btn btn-outline-dark"><span class="glyphicon glyphicon-menu-down"></span> COMMENTS</button>
+                <a href="index.html"><button type="button" class="btn btn-outline-dark custom-btn"><span class="glyphicon glyphicon-menu-left"></span> BACK</button></a>
+                <button type="button" onclick="getComments()" class="btn btn-outline-dark custom-btn"><span class="glyphicon glyphicon-menu-down"></span> COMMENTS</button>
+            </div>
+            <div id="add-comment">
+                <br>
+                <form action="/data" method="POST">
+                    <label for="display-name">Display Name:</label>
+                    <input type="text" id="display-name" name="display-name" value=" anonymous"><br>
+                    <textarea id="comment" name="comment" rows="5" cols="33">Add a comment...</textarea><br>
+                    <input class="btn btn-outline-dark custom-btn" type="submit" value="SUBMIT">
+                </form>
```

Comment:
Would have been nice if the client side changes were in a separate PR. But ok for now.